src/core/server/locales/en-US/errors.ftl

```diff
@@ -54,3 +54,4 @@ error-rateLimitExceeded = Rate limit exceeded.
 error-inviteTokenExpired = Invite link has expired.
 error-inviteRequiresEmailAddresses = Please add an email address to send invitations.
 error-passwordIncorrect = Password provided was incorrect.
+error-usernameAlreadyUpdated = You may only change your username once every 14 days.
```

Comment:
I don't know if this is possible, but it would be cool if this timespan could be templated into this error message. That way we could update a constant somewhere and update all the messages if they decide it should be, say, only every 30 days.